.travis.yml

```diff
@@ -17,10 +17,12 @@ matrix:
     - php: 5.5
       env: SYMFONY_VERSION=2.4.*
     - php: 5.5
-      env: SYMFONY_VERSION=2.*
+      env: SYMFONY_VERSION=2.5.*
+    - php: 5.5
+      env: SYMFONY_VERSION=2.7.*
 
 before_install:
-  - composer require symfony/routing:${SYMFONY_VERSION} --prefer-source
+  - composer require symfony/symfony:${SYMFONY_VERSION} --prefer-source
```

Comment:
this will however hide when we accidentally start to use other symfony parts that are not listed in composer.json - should we have one build where we just install without require?